routes/api/chromes.js

```diff
@@ -0,0 +1,50 @@
+const router = require('express').Router();
+
+const { Chrome }= require('../../databases/db');
+
+//endpoint para listar cromos
+router.get('/', async (req,res)=>{
+    //paginacion
+    const {page = 0, size = 10} = req.query;
+
+    let options = {
+        limit: +size,
+        offset: (+page) * (+size)
+    };
+    const users = await Chrome.findAll(options);
```

Comment:
```suggestion
    const stickers = await Chrome.findAll(options);
```
cambiar nombre de la variable a stickers

Reply:
listo papi

Reply:
@HopeAero la idea no seria solo aplicarlo puntualmente a 1 solo sitio xDDDD, si no, literal aplicarlo a todos los sitios donde se hace referencia a cromos (o en este caso a otra entidad de forma errada)